src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -379,6 +380,15 @@ public void periodic() {
     } else {
       goToPosition(desiredPosition);
     }
+
+    // If the grabber is set to open and it is safe to open, open the grabber. Otherwise, close it.
+    if ((armEncoder.getPosition() > Cal.Lift.GRABBER_CLOSED_ZONE_TOP_DEGREES
+            || armEncoder.getPosition() < Cal.Lift.GRABBER_CLOSED_ZONE_BOTTOM_DEGREES)
+        && grabberOpen) {
```

Comment:
Super minor but swap the order of grabberOpen and the other part? Marginally more readable that way IMO+